dev/io.openliberty.microprofile.internal_fat/fat/src/io/openliberty/microprofile/internal/test/suite/MPCompatibleTest.java

```diff
@@ -0,0 +1,346 @@
+        }
+    }
+
+    /**
+     * microProfile-3.3 plus mpGraphQL-1.0 and mpReactiveMessaging-1.0
+     * Should pass.
+     *
+     * mpGraphQL-1.0 currently produces some warnings that need to be investigated
+     * https://github.com/OpenLiberty/open-liberty/issues/15496
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.FULL)
+    @Ignore //TODO: re-enable when issue 15496 has been resolved
+    public void testMP33plusStandalone() throws Exception {
+        try {
+            server.setServerConfigurationFromFilePath("MP33plusStandalone.xml");
+            server.startServer();
+            runGetMethod(200, "/helloworld/helloworld", MESSAGE);
+        } finally {
+            server.stopServer("CWMOT0010W"); //CWMOT0010W: OpenTracing cannot track JAX-RS requests because an OpentracingTracerFactory class was not provided or client libraries for tracing backend are not in the class path.
+        }
+    }
+
+    /**
+     * microProfile-4.0 plus mpGraphQL-1.0
+     * mpReactiveMessaging-1.0 should be included once the following feature is completed
+     * https://github.com/OpenLiberty/open-liberty/issues/15440
+     *
+     * mpGraphQL-1.0 currently produces some warnings that need to be investigated
+     * https://github.com/OpenLiberty/open-liberty/issues/15496
+     *
+     * Should pass.
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.FULL)
+    @Ignore //TODO: re-enable when issue 15496 has been resolved
+    public void testMP40plusStandalone() throws Exception {
+        try {
+            server.setServerConfigurationFromFilePath("MP40plusStandalone.xml");
+            server.startServer();
+            runGetMethod(200, "/helloworld/helloworld", MESSAGE);
+        } finally {
+            server.stopServer("CWMOT0010W"); //CWMOT0010W: OpenTracing cannot track JAX-RS requests because an OpentracingTracerFactory class was not provided or client libraries for tracing backend are not in the class path.
+        }
+    }
+
+    private StringBuilder runGetMethod(int exprc, String requestUri, String testOut) throws IOException {
+        URL url = new URL("http://" + getHost() + ":" + getPort() + requestUri);
+        int retcode;
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+        try {
+            con.setDoInput(true);
+            con.setDoOutput(true);
+            con.setUseCaches(false);
+            con.setRequestMethod("GET");
+
+            retcode = con.getResponseCode();
+
+            InputStream is = con.getInputStream();
+            InputStreamReader isr = new InputStreamReader(is);
+            BufferedReader br = new BufferedReader(isr);
+
+            String sep = System.getProperty("line.separator");
+            StringBuilder lines = new StringBuilder();
+            for (String line = br.readLine(); line != null; line = br
+                                                                     .readLine())
```

Comment:
very odd format.

Reply:
um, that might have been Eclipse being special?!